internal/services/mssqlmanagedinstance/mssql_managed_instance_active_directory_administrator_resource_test.go

```diff
@@ -68,6 +68,42 @@ func (r MsSqlManagedInstanceActiveDirectoryAdministratorResource) template(data
 	return fmt.Sprintf(`
 %[1]s
 
+resource "azurerm_mssql_managed_instance" "test" {
+  name                = "acctestsqlserver%[2]d"
+  resource_group_name = azurerm_resource_group.test.name
+  location            = azurerm_resource_group.test.location
+
+  license_type       = "BasePrice"
+  sku_name           = "GP_Gen5"
+  storage_size_in_gb = 32
+  subnet_id          = azurerm_subnet.test.id
+  vcores             = 4
+
+  administrator_login          = "missadministrator"
+  administrator_login_password = "NCC-1701-D"
+
+  depends_on = [
+    azurerm_subnet_network_security_group_association.test,
+    azurerm_subnet_route_table_association.test,
+  ]
+
+  identity {
+    type = "SystemAssigned"
+  }
+
+  tags = {
+    environment = "staging"
+    database    = "test"
+  }
+
+  # changing azure_active_directory_administrator is ignored since the value of azure_active_directory_administrator is returned 
```

Comment:
do you mean is not returned?

Reply:
Hi @katbyte thanks for your time and feedback. I would like to explain that AAD could be enabled by specifying `azure_active_directory_administrator` in `azurerm_mssql_managed_instance` and specifying `azuread_authentication_only` to `true` in `azurerm_mssql_managed_instance_active_directory_administrator`(using `azurerm_mssql_managed_instance` is more flexible).

Adding `ignore_changes` to `azure_active_directory_administrator` is not because the API does not return, but because in the test case, `azurerm_mssql_managed_instance` is created first (without enabling AAD), and then `azurerm_mssql_managed_instance_active_directory_administrator` is created (enabling AAD by specifying `azuread_authentication_only` to true ). In this way, when TF refresh `azurerm_mssql_managed_instance` again after apply, the API returns the value of `azure_active_directory_administrator` (because AAD is enabled by `azurerm_mssql_managed_instance_active_directory_administrator`). If `ignore_changes` for `azure_active_directory_administrator` is not specified, the test case would fail with the following error. This is why I added `ignore_changes` to `azure_active_directory_administrator`. 


![image](https://github.com/user-attachments/assets/df2d93b9-ffdf-42c4-a37b-9a966be28fa3)